helix-core/src/test/java/org/apache/helix/integration/rebalancer/WagedRebalancer/TestWagedRebalance.java

```diff
@@ -135,6 +135,16 @@ protected void addInstanceConfig(String storageNodeName, int seqNo, int tagCount
 
   @Test
   public void test() throws Exception {
+    HelixDataAccessor dataAccessor = new ZKHelixDataAccessor(CLUSTER_NAME, _baseAccessor);
```

Comment:
Put this logic into the beforeClass() method as part of the initial configuration process? Or you shall remove this config once the test is done. Otherwise, the configuration might impact the other test cases unexpectedly.